test/userAgentFilterTests.ts

```diff
@@ -5,13 +5,17 @@ import * as assert from "assert";
 import { WebResource } from "../lib/webResource";
 import { MsRestUserAgentFilter } from "../lib/filters/msRestUserAgentFilter";
 import { Constants } from "../lib/util/constants";
+import { isNode } from "../lib/msRest";
```

Comment:
I've been trying to not import from `msRest`, but I'm not sure if it's important. The `isNode` variable lives inside a file in the `utils` folder.

Reply:
Oh, I see how it's kinda weird to use the "external" interface when we have access to the "internal" one.

Reply:
I think there's weird stuff going on here related to webpack `externals` config and msRest. I would like to be able to test the actual msRestBundle.js that's distributed, but basically imports that reach into the internal paths seem to result in redundant copies of stuff being included in the test bundle. Will look at it later down the line.